int32 command_id
float64 coord_x
float64 coord_y
float64 coord_z
float64 rot_roll
float64 rot_pitch
float64 rot_yaw
float64 gasp_diam
float64 dest_x
float64 dest_y
float64 dest_z
float64 dest_roll
float64 dest_pitch
float64 dest_yaw
int32 w_time
